tests/e2e/esm/index.test.js

```diff
@@ -396,34 +396,41 @@ describe('ANTRegistry', async () => {
 });
 
 describe('ANT', async () => {
-  const processId = 'aWI_dq1JH7facsulLuas1X3l5dkKuWtixcZDYMw9mpg';
+  const processId = 'YcxE5IbqZYK72H64ELoysxiJ-0wb36deYPv55wgl8xo';
   const ant = ANT.init({
     processId,
   });
 
   it('should be able to create ANTWriteable with valid signers', async () => {
     for (const signer of signers) {
-      const writeable = ANT.init({
+      const nonStrictAnt = ANT.init({
         processId,
         signer,
+        strict: false,
+      });
+      const strictAnt = ANT.init({
+        processId,
+        signer,
+        strict: true,
       });
 
-      assert(writeable instanceof AoANTWriteable);
+      assert(nonStrictAnt instanceof AoANTWriteable);
+      assert(strictAnt instanceof AoANTWriteable);
     }
   });
 
   it('should be able to get ANT info', async () => {
-    const info = await ant.getInfo({ processId });
+    const info = await ant.getInfo({ strict: true });
```

Comment:
Note: we don't need all these tests to enforce strict on the APIs. The main thing to test would be overriding the `strict` defined on the constructor to ensure the flexibility works as expected.

Reply:
[62c9140](https://github.com/ar-io/ar-io-sdk/pull/227/commits/62c914086b936b0c91cea757709941c1987f3ade)